app/javascript/application.js

```diff
@@ -1,6 +1,3 @@
 // Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
 import "@hotwired/turbo-rails"
 import "controllers"
-import Rails from "@rails/ujs"
-
-Rails.start()
```

Comment:
No, it wasn't interfering. I was reading through the network traffic and saw it pull in ujs.

Reply:
@robacarp I just discovered that elsewhere in our app, I added a `data: { confirm: "Are you sure?" },` to a form button and it didn't work. I tracked it down to this removal of UJS. ChatGPT is telling me that there is some baseline rails functionality such as this delete confirmation that are still handled by UJS and that it can coexist with stimulus. So I think we should add it back. Any thoughts?

Reply:
I'm not opposed, but I think my preference would be to just make it a stimulus controller.